src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1209,9 +1220,37 @@ namespace AirflowNetworkBalanceManager {
 			// Wind coefficient == Surface-Average does not need inputs of external nodes
 			CurrentModuleObject = "AirflowNetwork:MultiZone:ExternalNode";
 			AirflowNetworkNumOfExtNode = GetNumObjectsFound( CurrentModuleObject );
+			if ( AnyLocalEnvironmentsInModel ) {
+ 				CurrentModuleObject = "OutdoorAir:Node";
+ 				AirflowNetworkNumOfOutAirNode = GetNumObjectsFound( CurrentModuleObject );
+ 				AirflowNetworkNumOfExtNode += AirflowNetworkNumOfOutAirNode;
```

Comment:
@XuanLuoLBL It may write more compact by using object names, instead of reassigning CurrentModuleObject:

AirflowNetworkNumOfExtNode = GetNumObjectsFound("AirflowNetwork:MultiZone:ExternalNode";);  
if ( AnyLocalEnvironmentsInModel )   {
				AirflowNetworkNumOfOutAirNode = GetNumObjectsFound( "OutdoorAir:Node");  
  				AirflowNetworkNumOfExtNode += AirflowNetworkNumOfOutAirNode;  
}